usaspending_api/common/threaded_data_loader.py

```diff
@@ -90,9 +89,8 @@ def load_from_file(self, filepath, encoding='utf-8', remote_file=False):
             process.start()
 
         if remote_file:
-            aws_region = os.environ.get('USASPENDING_AWS_REGION')
-            with smart_open.smart_open(filepath, 'r', encoding=encoding, region_name=aws_region) as csv_file:
-                row_queue = self.csv_file_to_queue(csv_file, row_queue)
+            csv_file = codecs.getreader('utf-8')(filepath['Body'])
```

Comment:
filepath -> filepath['Body'] ?

Reply:
This was a fun time. Threaded Data Loader is used by `loadTas.py` (locally and remote support) and `load_reference_csv.py` (locally). Locally Threaded Data Loader is the same but after updating `loadTas` to use boto3 (see lines 24-25 of the new `loadTas.py`), the remote functionality needed to change I needed a way to take the csv on the bucket and have it be read by `DictReader`. This solution worked after testing pulling from S3 on my local machine. So in that case, `file_path` is a streaming body of the csv instead of the local `file_path` of the csv when `is_remote_file` is `true`.